### PR TITLE
DHFPROD-8541: Re-Add Run Flows Success and Error Messages

### PR DESCRIPTION
**Pull request by lucianaabc-ml**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests  

- ##### Reviewer:

- [x] Reviewed Tests
- [x] Added to Release Wiki

### CODE REVIEW

File: marklogic-data-hub-central/ui/e2e/cypress/integration/curation/run/runFlow.spec.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ describe("Run Tile tests", () => {
     cy.resetTestUser();
   });
 
-  it.skip("can create flow and add steps to flow, should load xml merged document and display content", {defaultCommandTimeout: 120000}, () => {
+  it("can create flow and add steps to flow, should load xml merged document and display content", {defaultCommandTimeout: 120000}, () => {
```
Comment (briantang) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/curation/run/runFlow.spec.tsx:28]: There's a line above this that was commented out due to the skipped test. Can we uncomment it?

Comment (briantang) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/curation/run/runFlow.spec.tsx:28]: There's a line above this that was commented out due to the skipped test. Can we uncomment it?

**Review by briantang — CHANGES_REQUESTED** [review]

**Review by briantang — APPROVED** [review]
Looks good now!

**Review by MSBrignoni — DISMISSED** [review]

**Review by briantang — CHANGES_REQUESTED** [review]
@lucianaabc-ml  There are several UI and e2e failures that all appear to be related. All of these tests still seem to be expecting the old modals when they should be asserting for the new status modal.

**Comment by briantang** [comment]
@lucianaabc-ml so close! Just two failures left.
1) In runFlows.spec, looks like it's missing a line to actually start running the flow before the modal appears.
2) In mappingWithCustomHeader.spec, it's trying to click a "back button" that isn't there. Maybe the solution is to remove that.

**Review by briantang — APPROVED** [review]

**Review by aniamy — APPROVED** [review]

**Pull request merged**